bigtable/examples/bigtable_hello_world.cc

```diff
@@ -0,0 +1,120 @@
+// Copyright 2018 Google Inc.
```

Comment:
New files should say "Google LLC" rather than "Google Inc."

No need to update all existing files at this time.

Reply:
Fixed.